odxtools/basicstructure.py

```diff
@@ -76,7 +76,12 @@ def get_static_bit_length(self) -> Optional[int]:
     def coded_const_prefix(self, request_prefix: bytes = b'') -> bytes:
         prefix = b''
         encode_state = EncodeState(
-            bytearray(prefix), parameter_values={}, triggering_request=request_prefix)
+            bytearray(prefix),
+            parameter_values={},
+            triggering_request=request_prefix,
+            cursor_byte_position=0,
+            cursor_bit_position=0,
+            origin_byte_position=0)
```

Comment:
In what case where `cursor_byte_position` and others would not be zero during EncodeState creation?

Reply:
this happens if a partially completed PDU is supposed to be modified. But I agree, this does not happen in the current code base, so these attributes ought to default to 0. fixed: c564c4a80df1acec67a845e2d3596ae7057ae5a0